ui/src/message_bar.tsx

```diff
@@ -22,6 +24,14 @@ export interface MessageBar {
 }
 
 const
+  css = stylesheet({
+    messageBar: {
```

Comment:
Can we add a comment here explaining what this CSS is doing / why this is required?

Reply:
Good catch @lo5. This CSS definitely deserves a comment. Will add.